src/api/config.json

```diff
@@ -1,16 +1,16 @@
 {
```

Comment:
I'd appreciate your thoughts on having a default config which 'just works'. I'd previously written it so that people cloning the repo would have to update the `config.json` first, to point it to their MQ instance (hence `example.com.invalid`, which is non-routable). Your proposed `config.json` will connect to a RabbitMQ instance with the default user running on localhost, which seems like a better idea 👍 - can we think of any downsides? (I'm trying to remember why I did it that way in the first place)